src/Pretix/OrderHelper.php

```diff
@@ -260,22 +275,30 @@ public function ensureWebhook(Client $client) {
   public function getSubEventAvailability(SubEvent $subEvent) {
     $event = $subEvent->getEvent();
     try {
-      $quotas = $this->pretixClient->getQuotas($event,
-        ['query' => ['subevent' => $subEvent->getId()]]);
+      $quotas = $this->pretixClient->getQuotas(
+            $event,
+            ['query' => ['subevent' => $subEvent->getId()]]
+        );
     }
     catch (\Exception $exception) {
-      throw $this->clientException('Cannot get quotas for sub-event',
-        $exception);
+      throw $this->clientException(
+            'Cannot get quotas for sub-event',
+            $exception
+            );
     }
 
     foreach ($quotas as $quota) {
       try {
-        $availability = $this->pretixClient->getQuotaAvailability($event,
-          $quota);
+        $availability = $this->pretixClient->getQuotaAvailability(
+              $event,
+              $quota
+          );
       }
       catch (\Exception $exception) {
-        throw $this->clientException('Cannot get quota availability',
-          $exception);
+        throw $this->clientException(
+              'Cannot get quota availability',
+              $exception
+                );
```

Comment:
The coding standards may not check/require this, but please be consistent in how code is indented. Your editor should have a “Reformat code” (or similar) function that can help with this.

Reply:
Fixed